application/pom.xml

```diff
@@ -530,5 +544,76 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>its</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>copy-data</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>copy</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>dev.ikm.tinkar.data</groupId>
+                                    <artifactId>tinkar-starter-data-reasoned</artifactId>
+                                    <classifier>pb</classifier>
+                                    <type>zip</type>
+                                    <overWrite>false</overWrite>
+                                    <outputDirectory>${user.home}/Solor</outputDirectory>
```

Comment:
This should not be in the home directory, but should be in the maven target directory.  This allows for cleanup of the artifacts, whereas somewhere in the home dir does not

Reply:
That was the initial implementation but somehow it failed during the CI builds. I'll return back to the initial implementation using the target directory and try it again, but after have organized have reconfigured the maven surefire and failsafe plugins.

Reply:
Instead, using a negated property for default activation worked. Moreover, this approach provides a simple, predictable way to skip the execution of the profile whenever necessary.